assets/base.css

```diff
@@ -2432,6 +2454,12 @@ input[type='checkbox'] {
   background: rgba(var(--color-foreground), 0.5);
 }
 
+@media screen and (max-width: 989px) {
```

Comment:
Do we have any conventions around mixing max-width and min-width media queries? Generally I would prefer something like this to avoid having to think about media queries in both directions:
```css
menu-drawer ~ .header__icons .header__icon--account {
  display: none;
}

@media screen and (min-width: 990px) {
  menu-drawer ~ .header__icons .header__icon--account {
    display: block;
  }
}
```
In other words, use min-width or max-width, but not both.

Reply:
I agree with you. However, unfortunately, we have a 50/50 split of using both min and max -width for media queries in base.css.
In other words there is no sense to change it in this PR. Maybe we should do it within the next tech debt or when we have time for this.